src/story.ts

```diff
@@ -286,11 +289,12 @@ export type ComponentAnnotations<
    * By defining them each component will have its tab in the args table.
    */
   subcomponents?: Record<string, TFramework['component']>;
-};
+}
 
 export type StoryAnnotations<
   TFramework extends AnyFramework = AnyFramework,
-  TArgs = Args
+  TArgs = Args,
+  TArgsAnnotations = Partial<TArgs>
```

Comment:
```suggestion
  TRequiredArgs = Partial<TArgs>
```

This is more intuitive to me, what do you think?

Reply:
Sounds good to me, what do you think @shilman?

Reply:
Works for me. Also discussed `TRemainingArgs` because these args are not necessarily required. But agree that "required" is the most readable/clear